HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -212,13 +212,25 @@ def customiseFor32066(process):
 
     return process
 
+def customiseFor32291(process):
```

Comment:
Why is this needed?
Inserting ```vetoEndcap = false``` should be taken care of by fillDescription, no?

Reply:
Unfortunately, PFBlockProducer doesn't utilize fillDescription yet. Something I am hoping to find a volunteer for.